common/static/common/js/discussion/views/discussion_thread_list_view.js

```diff
@@ -284,8 +224,7 @@
                 }
                 this.showMetadataAccordingToSort();
                 this.renderMorePages();
-                this.updateSidebar();
-                this.trigger('threads:rendered');
+                return this.trigger('threads:rendered');
```

Comment:
I agree with @bjacobel. My approach when converting CoffeeScript is to fix the code not to return values when a function is inconsistent, because clearly no callers can be depending upon the value. We should never need to add additional returns.